#VRML_SIM R2021a utf8
WorldInfo {
  basicTimeStep 16
  coordinateSystem "NUE"
  lineScale 0.04
}
Viewpoint {
  orientation 0.5783173175025412 0.5888653197923019 0.5646120043202897 4.227071288594503
  position -0.22730568718467947 2.194484447630438 1.1469435570231807
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  rotation 1 0 0 0
  floorSize 55 55
  floorTileSize 0.25 0.25
  floorAppearance Parquetry {
    type "dark strip"
    colorOverride 0.8 0.8 0.8
    IBLStrength 5
  }
}
SolidPipe {
  translation 12.6133 -8.88178e-16 -7.68088
  height 0.3
  radius 0.05
}
E-puck {
  translation 0.0852678 4.54082e-14 0.880818
  name "epuck1"
  controller "test_comms_position"
  supervisor TRUE
  battery [
    1800000, 1800000, 1
  ]
  turretSlot [
    LED {
      translation 0 0.001 -0.01
      children [
        Group {
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.74902 0.74902 0.74902
                roughness 1
                metalness 0
              }
              geometry DEF LED_SHAPE Sphere {
                radius 0.01
              }
            }
            PointLight {
              attenuation 0 0 1
              color 0 1 0
              intensity 0.5
              location 0 0.03 0
              on FALSE
              castShadows TRUE
            }
          ]
        }
      ]
      name "ledd"
      color [
        1 0 0
        0 1 0
        0 0 1
        1 1 0
        0 1 1
        1 0 1
        1 0.647059 0
        0.627451 0.12549 0.941176
        0.545098 0.411765 0.0784314
      ]
    }
    GPS {
      scale 0.01 0.01 0.01
    }
    Compass {
      rotation 0 1 0 1.5708003061004252
    }
    DEF EMITTER Emitter {
      name "bt_e"
      range 2
      channel 2
    }
    DEF RECEIVER Receiver {
      name "bt_r"
      channel 2
    }
    DEF EMITTER Emitter {
      translation 0 0 -0.04
      name "nfc_e"
      range 0.06
      channel 1
    }
    DEF RECEIVER Receiver {
      translation 0 0 -0.04
      name "nfc_r"
      channel 1
    }
  ]
}
E-puck {
  translation -0.4274 -3.33067e-16 0.516844
  rotation 0 1 0 -0.523595307179586
  name "epuck2"
  controller "test_comms_position"
  supervisor TRUE
  battery [
    1800000, 1800000, 2
  ]
  turretSlot [
    LED {
      translation 0 0.001 -0.01
      children [
        Group {
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.74902 0.74902 0.74902
                roughness 1
                metalness 0
              }
              geometry DEF LED_SHAPE Sphere {
                radius 0.01
              }
            }
            PointLight {
              attenuation 0 0 1
              color 0 0 1
              intensity 0.5
              location 0 0.03 0
              on FALSE
              castShadows TRUE
            }
          ]
        }
      ]
      name "ledd"
      color [
        1 0 0
        0 1 0
        0 0 1
        1 1 0
        0 1 1
        1 0 1
        1 0.647059 0
        0.627451 0.12549 0.941176
        0.545098 0.411765 0.0784314
      ]
    }
    GPS {
    }
    Compass {
    }
    DEF EMITTER Emitter {
      name "bt_e"
      range 2
      channel 2
    }
    DEF RECEIVER Receiver {
      name "bt_r"
      channel 2
    }
    DEF EMITTER Emitter {
      translation 0 0 -0.04
      name "nfc_e"
      range 0.06
      channel 1
    }
    DEF RECEIVER Receiver {
      translation 0 0 -0.04
      name "nfc_r"
      channel 1
    }
  ]
}
E-puck {
  translation -0.525168 3.13083e-14 1.52406
  rotation 0 1 0 0.2617989999999993
  name "epuck3"
  controller "test_comms_position"
  supervisor TRUE
  battery [
    1800000, 1800000, 2
  ]
  turretSlot [
    LED {
      translation 0 0.001 -0.01
      children [
        Group {
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.74902 0.74902 0.74902
                roughness 1
                metalness 0
              }
              geometry DEF LED_SHAPE Sphere {
                radius 0.01
              }
            }
            PointLight {
              attenuation 0 0 1
              color 0 0 1
              intensity 0.5
              location 0 0.03 0
              on FALSE
              castShadows TRUE
            }
          ]
        }
      ]
      name "ledd"
      color [
        1 0 0
        0 1 0
        0 0 1
        1 1 0
        0 1 1
        1 0 1
        1 0.647059 0
        0.627451 0.12549 0.941176
        0.545098 0.411765 0.0784314
      ]
    }
    GPS {
    }
    Compass {
    }
    DEF EMITTER Emitter {
      name "bt_e"
      range 2
      channel 2
    }
    DEF RECEIVER Receiver {
      name "bt_r"
      channel 2
    }
    DEF EMITTER Emitter {
      translation 0 0 -0.04
      name "nfc_e"
      range 0.06
      channel 1
    }
    DEF RECEIVER Receiver {
      translation 0 0 -0.04
      name "nfc_r"
      channel 1
    }
  ]
}
E-puck {
  translation 0.0508305 3.13083e-14 1.54851
  rotation 0 1 0 0.2617989999999993
  name "epuck4"
  controller "test_comms_position"
  supervisor TRUE
  battery [
    1800000, 1800000, 2
  ]
  turretSlot [
    LED {
      translation 0 0.001 -0.01
      children [
        Group {
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.74902 0.74902 0.74902
                roughness 1
                metalness 0
              }
              geometry DEF LED_SHAPE Sphere {
                radius 0.01
              }
            }
            PointLight {
              attenuation 0 0 1
              color 0 0 1
              intensity 0.5
              location 0 0.03 0
              on FALSE
              castShadows TRUE
            }
          ]
        }
      ]
      name "ledd"
      color [
        1 0 0
        0 1 0
        0 0 1
        1 1 0
        0 1 1
        1 0 1
        1 0.647059 0
        0.627451 0.12549 0.941176
        0.545098 0.411765 0.0784314
      ]
    }
    GPS {
    }
    Compass {
    }
    DEF EMITTER Emitter {
      name "bt_e"
      range 2
      channel 2
    }
    DEF RECEIVER Receiver {
      name "bt_r"
      channel 2
    }
    DEF EMITTER Emitter {
      translation 0 0 -0.04
      name "nfc_e"
      range 0.06
      channel 1
    }
    DEF RECEIVER Receiver {
      translation 0 0 -0.04
      name "nfc_r"
      channel 1
    }
  ]
}
E-puck {
  translation -0.614533 3.13083e-14 1.11127
  rotation 0 1 0 -0.523595307179586
  name "epuck5"
  controller "test_comms_position"
  supervisor TRUE
  battery [
    1800000, 1800000, 2
  ]
  turretSlot [
    LED {
      translation 0 0.001 -0.01
      children [
        Group {
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.74902 0.74902 0.74902
                roughness 1
                metalness 0
              }
              geometry DEF LED_SHAPE Sphere {
                radius 0.01
              }
            }
            PointLight {
              attenuation 0 0 1
              color 0 0 1
              intensity 0.5
              location 0 0.03 0
              on FALSE
              castShadows TRUE
            }
          ]
        }
      ]
      name "ledd"
      color [
        1 0 0
        0 1 0
        0 0 1
        1 1 0
        0 1 1
        1 0 1
        1 0.647059 0
        0.627451 0.12549 0.941176
        0.545098 0.411765 0.0784314
      ]
    }
    GPS {
    }
    Compass {
    }
    DEF EMITTER Emitter {
      name "bt_e"
      range 2
      channel 2
    }
    DEF RECEIVER Receiver {
      name "bt_r"
      channel 2
    }
    DEF EMITTER Emitter {
      translation 0 0 -0.04
      name "nfc_e"
      range 0.06
      channel 1
    }
    DEF RECEIVER Receiver {
      translation 0 0 -0.04
      name "nfc_r"
      channel 1
    }
  ]
}
